src/components/accordion/accordion.style.ts

```diff
@@ -1,56 +1,78 @@
+/* eslint-disable no-nested-ternary */
 import styled, { css } from "styled-components";
+import { space, margin, MarginProps } from "styled-system";
 
-import { space, margin } from "styled-system";
-
+import Box from "../box";
 import Icon from "../icon";
-import { baseTheme } from "../../style/themes";
-import ValidationIconStyle from "../../__internal__/validations/validation-icon.style";
 import addFocusStyling from "../../style/utils/add-focus-styling";
+import { baseTheme } from "../../style/themes";
 
-const StyledAccordionGroup = styled.div`
-  ${margin}
-`;
-
-export interface StyledAccordionContainerProps {
-  /** Toggles left and right borders, set to none when variant is subtle */
+interface StyledWrapperProps extends MarginProps {
```

Comment:
suggestion (non-blocking): I think it might be worth reusing some of the types from the component props interface as if we ever update them we need to update them here as well